niri-ipc/src/lib.rs

```diff
@@ -800,6 +800,22 @@ pub enum Transform {
     Flipped270,
 }
 
+impl Transform {
+    /// Return numeric identifier matching the `transform` enum of the `wl_output` interface.
+    pub fn to_wayland_id(self) -> u32 {
```

Comment:
This is not a "to wayland id", this is a "to mutter-specific private dbus interface id". So it should be left inside mutter_display_config.rs.

Reply:
The actual numbers do match wayland.xml, not sure why you think mutter could ever switch to different numbers :roll_eyes:

Reply:
It couldn't. But this code was written specifically to convert according to the mutter xml, and isn't used anywhere else